Source/WSTagView.swift

```diff
@@ -25,6 +29,46 @@ open class WSTagView: UIView, UITextInputTraits {
             setNeedsDisplay()
         }
     }
+    
+    open var imagePlaceholder = UIImage(systemName: "person.circle.fill") {
+        didSet {
+            if imageURL == nil {
```

Comment:
It's weird to be checking another variable inside a different variable didSet. I feel like we should separate out this logic into an `updateImage` or method or something. Also, didSet isn't called on init, right? Are we ever going to be setting the placeholder image to be something other than `person.circle.fill`?

Reply:
What I wanted to do with this is not restrict the user to use an image from a URL and allow him to set a `UIImage`. I think a better solution is to drop the `imageURL` and `imagePlaceholder` and allow the user to set the `UIImage`